kodilla-basic-tests/src/main/java/com/kodilla/abstracts/homework/Shape.java

```diff
@@ -0,0 +1,21 @@
+package com.kodilla.abstracts.homework;
+
+public abstract class Shape {
+
+    public int sideA;
+    public int sideB;
+    public int sideC;
+    public int height;
+    public int base;
+
+    public Shape (int sideA, int sideB, int sideC, int height, int base) {
+        this.sideA = sideA;
+        this.sideB = sideB;
+        this.sideC = sideC;
+        this.base = base;
+        this.height = height;
+    }
+
+    public abstract void getSurfaceArea();
```

Comment:
Nie void. Skoro metoda nazywa sie get... to oczekuje, ze cos dostane, tak? Zadne sout w tych metodach, tylko zwracanie policzonej wartosci